### PR TITLE
Adding new LTPA Configuration Unit Tests

### PR DESCRIPTION
**Pull request by mshah0722**
Added the following unit tests to the LTPA project:

1. fileMonitorRegistration_monitorValidationKeysDir
2.  fileMonitorRegistration_updateTriggerMbean
3.  fileMonitorRegistration_updateTriggerDisabled
4.  getMonitorInterval
5.  getMonitorValidationKeysDir
6.  getUpdateTrigger
7.  getValidationKeys
8.  getFirstValidationKeyFileName
9.  getFirstValidationKeyPassword
10.  getFirstValidationKeyValidUntilDate
11.  modified_monitorValidationKeysDirSet_everythingElseTheSame_keysReloaded
12.  modified_monitorValidationKeysDirSame_fileChanged_unregistersListenerAndCreatesKeys
13.  modified_fileAndMonitorValidationKeysDirChanged_unregistersListenerAndCreatesKeys
14. maskKeysPasswords_replacesPasswordWithMask

### CODE REVIEW

**Comment by mshah0722** [comment]
#libby
#build
#spawn.fullfat.buckets=com.ibm.ws.security.token.ltpa_fat

File: dev/com.ibm.ws.security.token.ltpa/src/com/ibm/ws/security/token/ltpa/internal/LTPAConfigurationImpl.java [hunk 1]
```diff
@@ -787,27 +787,35 @@ protected LTPAKeysChangeNotifier getLTPAKeysChangeNotifier() {
...
     }
 
+    /** {@inheritDoc} */
     @Sensitive
     @Override
     public List<Properties> getValidationKeys() {
         return validationKeys;
     }
 
+    protected List<Properties> testMaskKeysPasswords(List<Properties> originalList) {
```
Comment (Zech-Hein) [inline dev/com.ibm.ws.security.token.ltpa/src/com/ibm/ws/security/token/ltpa/internal/LTPAConfigurationImpl.java:815]: lets remove this getter and change `maskKeysPasswords` private to protected

**Review by Zech-Hein — APPROVED** [review]
LGTM

**Comment by mshah0722** [comment]
The build failed due to an unrelated error.

**Pull request merged**